microcosm_flask/swagger/schema.py

```diff
@@ -134,7 +140,7 @@ def build_schema(marshmallow_schema):
     required_fields = [
         field.dump_to or name
         for name, field in fields
-        if field.required and not field.allow_none
```

Comment:
I think this may break some existing calls in services (even though this looks like the right fix), so maybe notify people when publishing the new package?